docs/source/users/index.rst

```diff
@@ -45,3 +45,4 @@ Research Assistant or build additional tasks, a technical documentation can be f
    discarding-attempts
    sequencing
    configuration
+   memory-game
```

Comment:
While it's not strictly a research task, can we rename it research-task-memory for consistency with the other tasks?